src/main/resources/anniv5Resources/localization/eng/dimensiongatepack3/Cardstrings.json

```diff
@@ -0,0 +1,51 @@
+{
+  "${ModID}:Knighthood": {
+    "NAME": "Knighthood",
+    "DESCRIPTION": "Lose !M! HP. NL Choose a card in hand to Upgrade permanently. NL Exhaust.",
+    "EXTENDED_DESCRIPTION": [
+      "Upgrade permanently."
+    ]
+  },
+  "${ModID}:Inferno": {
+    "NAME": "Inferno",
+    "DESCRIPTION": "Deal !D! damage and apply 10 ${ModID}:Ignite to ALL enemies. NL Gain !M! ${ModID}:Ignite."
+  },
+  "${ModID}:Minniegun": {
+    "NAME": "Minniegun",
```

Comment:
Is this spelling a reference? Normally spelled Minigun

Reply:
It is, yes.